grpc/server/src/Mu/GRpc/Server.hs

```diff
@@ -29,6 +29,7 @@ module Mu.GRpc.Server
 , runGRpcAppTLS, TLSSettings
   -- * Convert a 'Server' into a WAI application
 , gRpcApp, gRpcAppTrans
+, gRpcMultipleApp, gRpcMultipleAppTrans
```

Comment:
As per @akshaymankar suggestion, we should consider exporting `MultipleServers` too?
```suggestion
, gRpcMultipleApp, gRpcMultipleAppTrans, MultipleServers(..)
```

Reply:
Also `gRpcMultipleServerHandlers`. Also, if that is being exposed it might be best to remove `ToMultipleServers` and instead provide functions to just translate tuples into `MultipleServers` and make these functions directly depend on `MultipleServers`